ansible-deploy-infrastructure-with-elastio/README.md

```diff
@@ -0,0 +1,72 @@
+## Deploy Infrastructure with Ansible and Elastio
+
+---
+### Author: Robert Saylor
+AWS Certified Cloud Practitioner
+
+Email: customphpdesign@gmail.com
+
+
+### Requirements
+- Ansible
+- boto2
+- python3
+
+### Pem Key
+Copy your PEM key to the pem directory and set the proper permission. The filename must match the key name in AWS.
+
+If your key name in AWS is my-ssh-key then the PEM filename would be my-ssh-key.pem
+
+### Operating Systems
+
+In this example we will cover Ubuntu 20.04 and AWS Linux 2.
+
+- Ubuntu 20.04 marketplace AMI "[ami-08d4ac5b634553e16](https://aws.amazon.com/marketplace/pp/prodview-iftkyuwv2sjxi)"
+- AWS Linux 2 marketplace AMI "[ami-0cff7528ff583bf9a](https://aws.amazon.com/marketplace/pp/prodview-wbr4b4xclx32c)"
+
+>> Before using any AMI documented it is best to go to AWS > Click on Launch EC2 > and search for the AMI there.
+
+### Playbook Parameters
+
+- ami : This should be a valid AMI on the AWS Marketplace
+- ssh_key: This is the name of your SSH key located on your AWS account
+- instance_type: Example: t3.medium
+- os_type: ubuntu | aws_linux (pass only one value)
+- security_group: The security group ID
+- vpc_subnet_id: The VPC subnet ID
+- region: The region to deploy the new EC2. Example: [us-east-1](https://docs.aws.amazon.com/general/latest/gr/rande.html)
+- profile: The IAM Profile with the permissions assigned
```

Comment:
It will be more clear to render the parameter names in a fixed with code font:


```suggestion
- `ami` : This should be a valid AMI on the AWS Marketplace
- `ssh_key`: This is the name of your SSH key located on your AWS account
- `instance_type`: Example: t3.medium
- `os_type`: ubuntu | aws_linux (pass only one value)
- `security_group`: The security group ID
- `vpc_subnet_id`: The VPC subnet ID
- `region`: The region to deploy the new EC2. Example: [us-east-1](https://docs.aws.amazon.com/general/latest/gr/rande.html)
- `profile`: The IAM Profile with the permissions assigned
```